gnmlibrarytool/templates/gnmlibrarytool/mediaview_storagerules.html

```diff
@@ -15,7 +15,7 @@
     </form>
     {% if user.is_superuser %}
         <div class="panel-header"><h2>Available Storage Rules</h2></div>
-        <form action="/gnmlibrarytool/addrule/" method="post">{% csrf_token %}
+        <form action="/gnmlibrarytool/item/add/" method="post">{% csrf_token %}
```

Comment:
this should be replaced with a {% url %} tag, to reference the name in the urls.py config